apps/cookbook/src/app/examples/icon-example/examples/default.ts

```diff
@@ -0,0 +1,39 @@
+import { Component } from '@angular/core';
+import { defaultIcons } from '@kirbydesign/designsystem';
+import { Color, ColorHelper } from '@kirbydesign/designsystem';
+
+const config = {
+  selector: 'cookbook-icon-default-example',
+  template: `<div>
+  <div class="icon-item-container" *ngFor="let icon of icons">
+    <div class="icon-item-inner-container">
+      <kirby-icon [name]="icon" [title]="icon" [themeColor]="color?.name"></kirby-icon>
+      <span class="icon-item-title">{{ icon }}</span>
+    </div>
+  </div>
+<div>
+
+  <p>Icons automatically inherit the value set via the <code>color</code> css property. You can experiment with various colors for the icons above here:</p>
+  <div class="color-options">
+    <button (click)="changeColor(color)" *ngFor="let color of colors" [ngClass]="color.name"></button>
```

Comment:
Some of the color options seem to be missing when comparing to https://cookbook.kirby.design/#/home/showcase/icon. In addition the labels have been removed.

Is this intended?

**Before**
![image](https://github.com/kirbydesign/designsystem/assets/150451/0cb6eb4b-569d-442b-aba9-ebd0dfdf72c2)

**After**
![image](https://github.com/kirbydesign/designsystem/assets/150451/28b57757-b87b-466a-b086-e93edf04d8ce)

Reply:
Yes, some of the colors made no sense as they would just dissapear or be very hard to seee, so I opted for a smaller set of colors as these are just examples anyway.